getcoverage.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+CPP_ETHEREUM_PATH=$(pwd)
+
+if [ ! -d "$CPP_ETHEREUM_PATH/build/test" ]; then
+   echo "You need to compile and build ethereum with cmake -DPROFILING option to the build dir!"
+   exit;	 
+fi
+
+OUTPUT_DIR="$CPP_ETHEREUM_PATH/build/test/coverage"
+TESTETH=$CPP_ETHEREUM_PATH/build  #/test/CMakeFiles/testeth.dir
+
+if which lcov >/dev/null; then
+    if which genhtml >/dev/null; then
+	lcov --directory $TESTETH --zerocounters
+	echo Running testeth...
+	$($CPP_ETHEREUM_PATH/build/test/testeth)
+	echo Prepearing coverage info...
+    else
+	echo genhtml not found
+	exit;
+    fi
+else
+    echo lcov not found
+    exit;
+fi
+
+echo Cleaning previous report...
+rm -r $OUTPUT_DIR/testeth
```

Please ignore error messages from `rm` commands